package.json

```diff
@@ -36,7 +36,7 @@
 	"dependencies": {
 		"axios": "^0.19.2",
 		"chokidar": "^3.3.0",
-		"timeline-state-resolver": "^3.19.0",
+		"timeline-state-resolver": "3.20.0-nightly-feature-smooth-casparcg-transitions-20200327-073602-c424637b.0",
```

Comment:
So is this one of those catch-22 scenarios where we won't have a proper version released until we've tested that it works here? IMO, using nightlies as dependencies for a (stable) master is not good practice.
Then again, the cutout tool is very much pre-release itself.

Reply:
yep, this is the only use-case (yet) where we can test it properly.